indico/web/client/js/react/components/ICSCalendarLink.module.scss

```
@@ -48,11 +48,21 @@
 .download-button:global(.ui.icon.button) {
   display: flex;
   align-items: center;
-  margin: 0;
+  margin: 0 0 0 0.5rem;
   height: 2.125rem;
 }
 
 .download-option {
   display: flex;
   align-items: center;
 }
+
+.popup:global(.ui.wide.popup) {
+  display: inline-flex;
+  max-width: 450px;
+}
+
+.popup-content {
```

We could have this one as just `.popup .content` and reference it as `styleName="content"` (since parent is already `popup`) as it seems clearer.
```suggestion
.popup .content {
```

---

Fixed :)